modules/10-user

```diff
@@ -6,5 +6,8 @@ source "${BASE_DIR}/framework.sh"
 
 user=${USER:-$(id -un)}
 hostname=${HOSTNAME:-$(hostname)}
+usercount=$(users | wc -w)
+users=$(who | awk '{print $1 $5}' | paste -sd ' ')
 
 print_columns "Logged as" "${user}@${hostname}"
+print_columns "Users ($usercount)" "$users"
```

Comment:
Could you please ensure that `$users` is an array and print it using `print_wrap` like in the [services module](https://github.com/bcyran/fancy-motd/blob/master/modules/34-services#L24)? This way it will be always correctly wrapped when there's a lot of items.